src/raftstore/coprocessor/dispatcher.rs

```diff
@@ -98,6 +112,37 @@ impl CoprocessorHost {
         }
     }
 
+    pub fn prepare_split_check(&self, region: &Region, engine: &DB) -> SplitCheckContext {
+        let mut ob_ctx = ObserverContext::new(region);
+        let mut split_ctx = SplitCheckContext::default();
+        for entry in &self.registry.observers {
+            entry
```

Comment:
It may stop early.

Reply:
It cant stop early, there may be more than one split check observer.